VotingApplication/VotingApplication.Web.Tests/Controllers/PollResultControllerTests.cs

```diff
@@ -158,6 +158,32 @@ public void NamedVoting_True_ReturnsNotNullForVoterName()
             }
 
             [TestMethod]
+            [ExpectedHttpResponseException(HttpStatusCode.BadRequest)]
+            public void Election_WithNoVote_ThrowsBAdRequest()
```

Comment:
The name of this test doesn't really convey what it's doing. Is it no ballot token being passed in the X-Header, or something else?